modules/connecttest.lua

```diff
@@ -192,37 +197,40 @@ function Test:initHMI()
       end
     end
   end
-
-  EXPECT_HMIEVENT(events.connectedEvent, "Connected websocket")
-  :Do(function()
-      registerComponent("Buttons", {"Buttons.OnButtonSubscription"})
-      registerComponent("TTS")
-      registerComponent("VR")
-      registerComponent("BasicCommunication",
-        {
-          "BasicCommunication.OnPutFile",
-          "SDL.OnStatusUpdate",
-          "SDL.OnAppPermissionChanged",
-          "BasicCommunication.OnSDLPersistenceComplete",
-          "BasicCommunication.OnFileRemoved",
-          "BasicCommunication.OnAppRegistered",
-          "BasicCommunication.OnAppUnregistered",
-          "BasicCommunication.PlayTone",
-          "BasicCommunication.OnSDLClose",
-          "SDL.OnSDLConsentNeeded",
-          "BasicCommunication.OnResumeAudioSource"
-        })
-      registerComponent("UI",
-        {
-          "UI.OnRecordStart"
-        })
-      registerComponent("VehicleInfo")
-      registerComponent("Navigation",
-        {
-          "Navigation.OnAudioDataStreaming",
-          "Navigation.OnVideoDataStreaming"
-        })
-    end)
+  if config.remoteConnection.enabled then
```

Comment:
@AStarodub I would propose next code instead to avoid duplication:
```
  local connectEventExp = EXPECT_HMIEVENT(events.connectedEvent, "Connected websocket")
  if not config.remoteConnection.enabled then
    connectEventExp:Do(function()
        registerComponent("Buttons", {"Buttons.OnButtonSubscription"})
        registerComponent("TTS")
        registerComponent("VR")
        registerComponent("BasicCommunication",
          {
            "BasicCommunication.OnPutFile",
            "SDL.OnStatusUpdate",
            "SDL.OnAppPermissionChanged",
            "BasicCommunication.OnSDLPersistenceComplete",
            "BasicCommunication.OnFileRemoved",
            "BasicCommunication.OnAppRegistered",
            "BasicCommunication.OnAppUnregistered",
            "BasicCommunication.PlayTone",
            "BasicCommunication.OnSDLClose",
            "SDL.OnSDLConsentNeeded",
            "BasicCommunication.OnResumeAudioSource"
          })
        registerComponent("UI",
          {
            "UI.OnRecordStart"
          })
        registerComponent("VehicleInfo")
        registerComponent("Navigation",
          {
            "Navigation.OnAudioDataStreaming",
            "Navigation.OnVideoDataStreaming"
          })
      end)
  end
```

Reply:
good suggestion, done